lab-3/lab3.m

```diff
@@ -0,0 +1,70 @@
+% Clear workspace
```

Comment:
@erla2996 This script seems to get fairly good results, let me know what you think